src/test/resources/features/ifs/fieldCollectionsIFS/VATFCBreathingSpace.feature

```diff
@@ -0,0 +1,188 @@
+      | paymentAmount | paymentDate |
+      | 100000        | 2021-06-01  |
+    And the fc vat customer has breathing spaces applied
+      | debtRespiteFrom | debtRespiteTo |
+      | 2021-11-01      | 2021-12-01    |
+    And the two debtItems put together
+    When the debt item is sent to the fc vat ifs service
+    Then the fc vat ifs service wilL return a total debts summary of
+      | combinedDailyAccrual | unpaidAmountTotal |
+      | 0                    | 400000            |
+    And the 1st fc vat debt summary will contain
+      | debtItemChargeId | interestDueDailyAccrual | interestRate |
+      | 123              | 0                       | 0.0          |
+
+
+  Scenario: Breathing space for no interest bearing debt with payments.
+    Given a fc vat debt item
+      | debtItemChargeId | originalAmount | periodEnd  | interestRequestedTo | interestIndicator |
+      | 123              | 500000         | 2022-04-01 | 2021-01-01          | Y                 |
+    And the fc vat debt item has payment history
+      | paymentAmount | paymentDate |
+      | 100000        | 2021-06-01  |
+    And the fc vat customer has breathing spaces applied
+      | debtRespiteFrom | debtRespiteTo |
+      | 2021-11-01      | 2021-12-01    |
+    And the two debtItems put together
+    When the debt item is sent to the fc vat ifs service
+    Then the fc vat ifs service wilL return a total debts summary of
+      | combinedDailyAccrual | unpaidAmountTotal |
+      | 28                   | 400000            |
+    And the 1st fc vat debt summary will contain
+      | debtItemChargeId | interestDueDailyAccrual | interestRate |
+      | 123              | 28                      | 2.6          |
+
+
+  Scenario: Breathing space for interest bearing debt with no payments.
+    Given a fc vat debt item
+      | debtItemChargeId | originalAmount | periodEnd  | interestRequestedTo | interestIndicator |
+      | 123              | 500000         | 2022-04-01 | 2022-01-04          | Y                 |
+    And the fc vat debt item has no payment history
+    And the fc vat customer has breathing spaces applied
+      | debtRespiteFrom | debtRespiteTo |
+      | 2021-11-01      | 2021-12-01    |
+    And the two debtItems put together
+    When the debt item is sent to the fc vat ifs service
+    Then the fc vat ifs service wilL return a total debts summary of
+      | combinedDailyAccrual | unpaidAmountTotal |
+      | 37                   | 500000            |
+    And the 1st fc vat debt summary will contain
+      | debtItemChargeId | interestDueDailyAccrual | interestRate |
+      | 123              | 37                      | 2.75         |
+
+
+  Scenario: Non interest bearing debt should not have breathing space applied
+    Given a fc vat debt item
+      | debtItemChargeId | originalAmount | periodEnd  | interestRequestedTo | interestIndicator |
+      | 123              | 500000         | 2022-04-01 | 2023-03-01          | N                 |
```

Comment:
interestRequestedTo should be before debtRespiteTo?

Reply:
uodated